server/errorHandler.ts

```diff
@@ -19,6 +19,9 @@ export default function createErrorHandler(production: boolean) {
 
     res.status(error.status || 500)
 
+    if (res.statusCode === 404) {
```

Comment:
Pretty sure the error handler at line 50 in `/server/app.ts` should take care of this?

Reply:
Hi @psoleckimoj. 

This line ensures that the any 404 errors are redirected to the custom SLM error page (https://github.com/ministryofjustice/hmpps-send-legal-mail-staff-ui/pull/6/files#diff-0ea50547ec1772ad781f22a66d858970793cbf1d97fd841f6e9beffe8e99b3c0)

Shows a generic error if I take this line out.